src/libs/utils.js

```diff
@@ -254,3 +254,8 @@ export const withBusyState = _.curry((setBusy, fn) => async (...args) => {
     setBusy(false)
   }
 })
+
+// Performs the given effect, but only on component mount
+export const useOnMount = fn => {
```

Comment:
Note: I shortened the name to `useOnMount`, which still seems like it communicates the intent. Also, apparently splitting it out like this also removes the lint warning, so it's not necessarily to silence it.

Reply:
I wonder if we still should, in case the lint rule gets clever enough to add it again, just to communicate intent? Or is there an explicit exception in the rule?

Reply:
I think it's just because it can't introspect into the `fn` because it's not inline. Between then name and the comment, I think we're good on intent, and it's probably a plus to avoid eslint overrides if possible.

Reply:
I wish I could weigh in on the naming, but I still haven't digested `useEffect`. Gut feeling, though, is to wonder if the word `use` is useful in the name of our utility function? Intuitively, it feels a little clumsy given the description of what the function is for. I feel like `use` communicates how the is _implemented_, but that detail does not usefully communicate how it should be _used_.

Reply:
The convention React is pushing is that any hook should be named `useX`.

Reply:
https://reactjs.org/docs/hooks-faq.html#what-exactly-do-the-lint-rules-enforce